sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -1561,9 +1562,9 @@ object CodeGenerator extends Logging {
   private def logGeneratedCode(code: CodeAndComment): Unit = {
     val maxLines = SQLConf.get.loggingMaxLinesForCodegen
     if (Utils.isTesting) {
-      logError(s"\n${CodeFormatter.format(code, maxLines)}")
```

Comment:
@gengliangwang 
Do we really need to change this to `log"...."`? 
IMO, maybe maintaining the original feeling is more suitable, which will be `very``very` `large`.

Reply:
Yeah, let's skip this one. +1 with @pan3793 @panbingkun